rero_ils/templates/rero_ils/frontpage.html

```diff
@@ -125,7 +125,9 @@ <h3>Il progetto</h3>
           <h3>{{ _('Organisations\' catalogues') }}</h3>
           <ul class="list-unstyled">
             {% for org in organisations %}
-            <li><a href="{{ url_for('rero_ils.index_with_view_code', viewcode=org.code) }}">{{ org.name }}</a></li>
+              {% if org.code != 'cypress' %}
```

Comment:
I'm not ultra fan about this string check here. Maybe it will be better to call a function on `organisation` to know if the organisation is a test one. Something like this in the template :
``` {% if !org.is_test_organisation %} ```
and in `Organisation` class :
```
...
@property
def is_test_organisation(self):
    if self.code == 'cypress':
        return False
    if self.other_test_could_be_placed_here():
        return False
    return True
```

Reply:
Done